### ISSUE TITLE
BUCKET_DIR_PATH should be overwritten to stellarCoreRunner.storagePath

### ISSUE THREAD

**Issue by bartekn** [stellar/go#3699]
### What version are you using?

2.4.1

### What did you do?

If `--captive-core-config-append-path` file contains `BUCKET_DIR_PATH` it gets propagated to generated `stellar-core.cfg` file used by Captive-Core.

### What did you expect to see?

`BUCKET_DIR_PATH` is ignored and the final `BUCKET_DIR_PATH` is set to `stellarCoreRunner.storagePath`+`/buckets` (generated using `--captive-core-storage-path`). Or `BUCKET_DIR_PATH` is removed from the final config file completely (then defaults to `buckets` in working dir).

### What did you see instead?

When running multiple Stellar-Cores (ex. parallel reingestion) `BUCKET_DIR_PATH` is reused causing:
```
Got an exception: unable to flock file: /BUCKET_DIR_PATH/stellar-core.lock (Resource temporarily unavailable). This can be caused by access rights issues or another stellar-core process already running
```

**Comment by 2opremio** [comment]
I think we should error (or warn) if `BUCKET_DIR_PATH` is included, not just silently replace it.

**Comment by bartekn** [comment]
Closed in #3707.
